crates/cargo-contract/src/cmd/info.rs

```diff
@@ -67,7 +70,22 @@ impl InfoCommand {
 
             match info_result {
                 Some(info_result) => {
-                    InfoCommand::print_and_format_contract_info(info_result);
+                    let output_type = match self.output_json {
+                        true => OutputType::Json,
+                        false => OutputType::HumanReadable,
+                    };
```

Comment:
It is redundant to define and construct this enum. can just do `if self.output_json`.

I can see that it is copied from `build`, but in that case it is passed down to other commands via `ExecuteArgs` so makes more sense. Whereas in this case the conditional is evaluated inline.

Reply:
Yep I hesitate to do this in this simple way